contracts/cw721-base/src/lib.rs

```diff
@@ -20,52 +23,116 @@ use cosmwasm_std::Empty;
 pub const CONTRACT_NAME: &str = "crates.io:cw721-base";
 pub const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+#[deprecated(
+    since = "0.19.0",
+    note = "Please use `DefaultOptionNftExtension` instead"
+)]
+pub type Extension = DefaultOptionalNftExtension;
+
+/// `cw721-base` with `TNftExtension` and `TCollectionExtension` allowing contract handling with:
```

Comment:
I'm not sure if there is supposed to be such a long comment? Perhaps it could go in a readme?